ui/narrative/methods/view_proteinstructures/spec.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name" : "View ProteinStructures",
+  "ver" : "1.0.0",
+  "authors" : [ ],
+  "contact" : "http://kbase.us/contact-us/",
```

Comment:
That page isn't found, please set it to https://www.kbase.us/support/

Reply:
I have found 57 cases of `contact: "help@kbase.us"` in addition to several `contact: "[help@kbase.us](http://kbase.us/contact-us/)"` when I was correcting the spelling of `visble`.